doc/odoc_for_authors.mld

```diff
@@ -32,36 +32,36 @@ v}
 - {{:https://github.com/ocaml-doc/voodoo}Voodoo} is the driver used to create
   the docs for {{:https://ocaml.org/packages}the OCaml website}.
 
-{1:interfaces Documenting your interfaces}
+{1:interfaces Documenting Your Interfaces}
 
-Odoc is built to produce documentation for your {e libraries}, and the unit of
+[odoc] is built to produce documentation for your {e libraries}, and the unit of
 organisation is the {e module}. Documentation is written by putting
-{{!special_comments}special comments} into the source of the {e module interface} or
-less commonly your {e module implementation}. 
+{{!special_comments}special comments} into the source of the {e module interface} or,
+less commonly, your {e module implementation}. 
 
 For the HTML output, [odoc] will produce one page for each module, module type,
-class and class type. This includes any submodules, so if a module [A] contains
-module [B] then two HTML pages will be created, and [A/index.html] will have a
+class, and class type. This includes any submodules, so if a module [A] contains
+module [B] then two HTML pages will be created. [A/index.html] will have a
 link to [A/B/index.html] at the point that module [B] is defined. So each
-module, module type, class and class type should be documented keeping in mind
+module, module type, class, and class type should be documented, keeping in mind
 that it will end up as a single page with a table of contents. 
 
 For each module, [odoc] will describe all of the values, types, module types,
-classes, modules and all other elements found in module signatures, documenting
+classes, modules, and all other elements found in module signatures, documenting
 the details relevant to the OCaml type system along with any documentation
 written in specially formatted comments in the source code. For any element that
 references any other, for example a [value] that has a [type], [odoc] will
 create links such that clicking them will link to the definitions.
```

Comment:
```suggestion
the details relevant to the OCaml type system along with any documentation
written in specially formatted comments in the source code. These comments, delineated by `(** *)`, are called docstrings. For any element that
references another, for example a [value] that has a [type], [odoc] will
create links that point to the definitions.
```

Reply:
So many changes... the entire section with proposed changes:

"For each module, [odoc] will describe all of the values, types, module types,
classes, modules, and other elements found in module signatures. Their types or descriptions are documented along with any comments delineated by `(** ... *), known as docstrings, found in the source code. For any element that references another, for example a [value] that has a [type], [odoc] will
create links that point to the definitions."